Sources/LanguageServerProtocol/Requests/InlayHintsRequest.swift

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Request for inline annotations to be displayed in the editor.
+///
+/// - Parameters:
+///   - textDocument: The document for which to provide the inlay hints.
+///
+/// - Returns: InlayHints for the entire document
+public struct InlayHintsRequest: TextDocumentRequest, Hashable {
```

Comment:
Please add `LSP Extension` to the doc comment -- see `SymbolInfoRequest` for an example.  If there's a stable place to link to for information about the proposed spec, that might also make sense to add.

Reply:
Sure, I've linked the PR and noted the commit of the current proposal state.